junit_tests/testTimeDelta.java

```diff
@@ -0,0 +1,423 @@
+import org.junit.Test;
+import org.python.stdlib.datetime.TimeDelta;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.python.types.Bool.getBool;
+import static org.python.types.Int.getInt;
+
+public class testTimeDelta {
+    private final Map<java.lang.String, org.python.Object> empty_kwargs = Collections.emptyMap();
+
+    private TimeDelta createDelta(int... values) {
+        return createDelta(empty_kwargs, values);
+    }
+
+    private TimeDelta createDelta(Map<java.lang.String, org.python.Object> kwargs, int... values) {
+        org.python.Object[] args = new org.python.Object[values.length];
+
+        for (int i = 0; i < values.length; i++) {
+            args[i] = getInt(values[i]);
+        }
+
+        return new TimeDelta(args, kwargs);
+    }
+
+    private TimeDelta createDelta(float... values) {
+        org.python.Object[] args = new org.python.Object[values.length];
+
+        for (int i = 0; i < values.length; i++) {
+            args[i] = new org.python.types.Float(values[i]);
+        }
+
+        return new TimeDelta(args, empty_kwargs);
+    }
+
+    @Test
+    public void testConstructorOneArguments() {
+        TimeDelta delta = createDelta(1);
+
+        assertEquals(getInt(1), delta.days);
+        assertEquals(getInt(0), delta.seconds);
+        assertEquals(getInt(0), delta.microseconds);
+    }
+
+    @Test
+    public void testConstructorTwoArguments() {
+        TimeDelta delta = createDelta(1, 2);
+
+        assertEquals(getInt(1), delta.days);
+        assertEquals(getInt(2), delta.seconds);
+        assertEquals(getInt(0), delta.microseconds);
+    }
+
+    @Test
+    public void testConstructorThreeArguments() {
+        TimeDelta delta = createDelta(1, 2, 3);
+
+        assertEquals(getInt(1), delta.days);
+        assertEquals(getInt(2), delta.seconds);
+        assertEquals(getInt(3), delta.microseconds);
+    }
+
+    @Test
+    public void testConstructorBadArguments() {
+        org.python.Object[] args = new org.python.Object[]{getBool(false)};
```

Comment:
Again, could use `org.python.types.Bool.FALSE` instead :smiley:

Reply:
Fixed all instances of `getBool(false)` and `getBool(true)` to `org.python.types.Bool.FALSE` and `org.python.types.Bool.TRUE` 😃